apps/hubble/src/rpc/httpServer.ts

```diff
@@ -594,12 +594,17 @@ export class HttpAPIServer {
         const pageOptions = getPageOptions(request.query);
 
         let relatedObjectRef;
-        if (ref_type == ObjectRefTypes.FID) {
-          relatedObjectRef = ObjectRef.create({ fid: object_ref_fid });
-        } else if (object_ref_network && object_ref_fid && object_ref_hash) {
+        let refType = Number.parseInt(ref_type);
+        let refDirection = Number.parseInt(ref_direction) as RefDirection;
+        let fid = Number.parseInt(object_ref_fid);
+        let network = Number.parseInt(object_ref_network) as FarcasterNetwork;
+        if (refType == ObjectRefTypes.FID) {
+          relatedObjectRef = ObjectRef.create({ type: ObjectRefTypes.FID, fid });
+        } else if (object_ref_network && fid && object_ref_hash) {
           relatedObjectRef = ObjectRef.create({
-            network: object_ref_network as FarcasterNetwork,
-            fid: object_ref_fid,
+            type: ObjectRefTypes.OBJECT,
+            network: network,
```

Comment:
not specifying the type of `ObjectRef` was causing it to always be created with the default 0 type, which is FID rather than OBJECT.